AssistantV2.playground/Pages/Message.xcplaygroundpage/Contents.swift

```diff
@@ -0,0 +1,54 @@
+//:## Message
+
+import PlaygroundSupport
+
+// Enable support for asynchronous completion handlers
+PlaygroundPage.current.needsIndefiniteExecution = true
+
+import AssistantV2
+
+let assistant = setupAssistantV2()
+var assistantID = WatsonCredentials.AssistantV2ID
+
+// Setup
+
+var sessionID = ""
+
+assistant.createSession(assistantID: assistantID) {
```

Comment:
Could we move the setup and cleanup code to the Helpers file, just to minimize the clutter on this page?

Reply:
I thought we should do that too, but I did it this way to match how it's done in other playgrounds (like AssistantV1 Synonyms). If you prefer changing all playgrounds to move setup/teardown code to the Helpers.swift, I'm onboard with that.

Reply:
Yes, let's put the setup/cleanup code in helpers in all the playgrounds. I think this is something I started doing later and didn't go back to fix in the earlier ones.

Reply:
Ok. I'll merge this PR as-is, and start another PR to move the setup/teardown code for all of the services.